src/main/java/org/folio/fqm/service/EntityTypeService.java

```diff
@@ -115,6 +115,14 @@ public ColumnValues getFieldValues(UUID entityTypeId, String fieldName, @Nullabl
       .findFieldDefinition(new FqlField(fieldName), entityType)
       .orElseThrow(() -> new FieldNotFoundException(entityType.getName(), fieldName));
 
+    if (field.getValues() != null) {
+      return getFieldValuesFromEntityTypeDefinition(field, searchText);
+    }
+
+    if (field.getValueSourceApi() != null) {
+      return getFieldValuesFromApi(field, searchText);
+    }
+
     if (field.getSource() != null) {
       if (field.getSource().getType() == SourceColumn.TypeEnum.ENTITY_TYPE) {
```

Comment:
If the source type isn't defined, it defaults to entity-type, so it was always ending up trying to look up values using the DB instead of the API. Whoops!